src/domain/accounts/limits-volume.ts

```diff
@@ -0,0 +1,145 @@
+import {
+  AmountCalculator,
+  paymentAmountFromNumber,
+  WalletCurrency,
+  ZERO_CENTS,
+} from "@domain/shared"
+import { addAttributesToCurrentSpan } from "@services/tracing"
+
+const calc = AmountCalculator()
+
+export const calculateLimitsInUsd = async ({
+  limitName,
+  limitAmount,
+  priceRatio,
+
+  walletVolumes,
+}: {
+  limitName:
```

Comment:
Anther galoy - enum. Don't like the hard coded Strings.

Reply:
Done: https://github.com/GaloyMoney/galoy/pull/1953/commits/934dc92450970208dd31a69aa9fed0797b822c61

I tried to do a bigger cleanup around this here (https://github.com/GaloyMoney/galoy/compare/a3583194056d0ac9151b0e33e74c8e397e31e472...0deae6bbe2d86c7ce10d1d3d21fa3617e83f984f) but the mocks for the limits test inexplicably broke.